### DISCUSSION TITLE
pfELK on existing openstack cluster

### DISCUSSION THREAD

**Discussion by mephmanx** [pfelk/pfelk#333] [Q&A]
I love the look of this project and am looking at working it into my existing openstack cluster.  PFSense is used as my cloud edge router and openstack contains an ELK stack.  I am looking at working this into my setup.  Anyone have advice?  Im not sure I am able to use the scripts as my infra already exists and is fully configured.  Id be looking to import the dashboards and get PFSense configured to forward logs to logstash/syslog.

**Comment by a3ilson** [comment]
@mephmanx - The scripts would not likely work but I'd be happy to help. Leveraging containers (docker) might be feasible alternative - how are you planning to install?
